packages/manager/apps/cloud/client/app/app.js

```diff
@@ -8,6 +8,7 @@ import { Environment } from '@ovh-ux/manager-config';
 import ovhManagerCore from '@ovh-ux/manager-core';
 import ngAtInternet from '@ovh-ux/ng-at-internet';
 import ngAtInternetUiRouterPlugin from '@ovh-ux/ng-at-internet-ui-router-plugin';
+import ngOvhAccountMigrationComponents from '@ovh-ux/manager-account-migration';
```

Comment:
Not sure the `ngOvh` prefix is necessary here, something like `ovhManager` instead maybe

Reply:
Renamed to `ovhManagerAccountMigration`. Thanks :)